web-client/src/presenter/actions/CaseInventoryReport/setCustomCaseInventoryFiltersAction.test.ts

```diff
@@ -334,5 +334,43 @@ describe('setCustomCaseInventoryFiltersAction', () => {
         'Roger Waters',
       ]);
     });
+
+    it('should negate the high priority when pass in as props', async () => {
```

Comment:
```suggestion
    it('should negate the highPriority filter when passed in as props', async () => {
```
Feel free to ignore!